src/SwedbankPay.Sdk.Infrastructure/PaymentInstruments/Trustly/TrustlyPayment.cs

```diff
@@ -1,12 +1,12 @@
 ﻿using SwedbankPay.Sdk.Extensions;
-
+using SwedbankPay.Sdk.Payments.TrustlyPayments;
 using System;
 using System.Net.Http;
 using System.Threading.Tasks;
 
-namespace SwedbankPay.Sdk.Payments.TrustlyPayments
+namespace SwedbankPay.Sdk.Payments
 {
-    public class TrustlyPayment
+    public class TrustlyPayment : ITrustlyPayment
```

Comment:
Codacy found an issue: [This class can't be instantiated; make its constructor 'public'.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)